Base/res/html/misc/welcome.html

```diff
@@ -211,7 +211,7 @@ <h2>Image Formats</h2>
             <li><a href="pngsuite_siz_png.html">pngsuite odd sizes test</a></li>
```

Comment:
Commit typo: Ladybirdy

Reply:
That's just compensation for [`LADBIRD`](https://github.com/SerenityOS/serenity/pull/18682#issue-1698677221) xD